File: test_as2js/TestTdd.js
var assert = require('assert');

/**
 * @author Ethan Kennerly
 */
suite('TestTdd', function()
{
  test('OneEqualsTwo', function()
  {
    var one = 1;
    assert.equal(1, one);
    assert.equal(2, one);
  });
});

File: test_as2js/TestTdd.cs
using NUnit.Framework;
namespace /*<com>*/Finegamedesign.As2js
{[TestFixture] public class TestTdd
    {
        [Test] public void OneEqualsTwo()
        {
            int one = 1;
            Assert.AreEqual(1, one);
            Assert.AreEqual(2, one);
        }
    }
}